frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts

```diff
@@ -10,3 +11,6 @@ export const getVMIApiPath = (vmi: VMIKind) =>
   `${VirtualMachineInstanceModel.apiVersion}/namespaces/${getNamespace(vmi)}/${
     VirtualMachineInstanceModel.plural
   }/${getName(vmi)}`;
+
+export const getVMINetworks = (vmi: VMIKind) => _.get(vmi, 'spec.networks', []);
```

Comment:
please add return type ie `VMIKind['spec]['networks']`, same for `getVMIDisks`

Reply:
Per discussion in other PR, I have replaced the `_.get()` by direct access.

Reply:
nice